web/oauth.tsx

```diff
@@ -31,11 +31,13 @@ function RenderOAuth() {
                 <code>{errcode}</code> {error}
             </p>
         </>;
-    } else if (result === 'pending') {
+    // Pending / update are mostly the same thing. Pending means a new app install, update means updating the existing app install.
+    } else if (result === 'pending' || result === 'update') {
         return <>
-            <h1>Your connection to { serviceName } {oauthKind} is pending.</h1>
+            <h1>The connection to your { serviceName } {oauthKind} is pending.</h1>
```

Comment:
Curious: why are there spaces around `serviceName` but not `oauthKind` or `result`?